### DISCUSSION TITLE
Data Start values don't work after changing series

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#2151] [Q&A]
Hello,

Site repo: https://github.com/christoph-LSN/MT_Site
data repo: https://github.com/christoph-LSN/IM_Data
metadata file: https://github.com/christoph-LSN/IM_Data/blob/develop/meta/1-2-5.md
example indicator: https://christoph-lsn.github.io/MT_Site/1-2-5/

The indicator shows people living in lower saxony by country of origin.
data start values are series "Alle Länder" and the continets.

Chaning series from "Alle Länder" to "Ukraine" or  to "Top 8 Asylherkunftsländer" and back to "Alle Länder" does not show the continents but "Andorra"

any ideas @brockfanning

**Comment by christoph-LSN** [comment]
Any Ideas @MoBosse @LucyGwilliamAdmin ?

**Comment by brockfanning** [comment]
Hey @christoph-LSN yes this seems like a bug. The platform is supposed to behave as if changing the series behaves the same as if you had reloaded the page. We'll look into it.

**Comment by christoph-LSN** [comment]
@brockfanning Thank you!
